.docksal/docksal.env

```diff
@@ -1,28 +1,23 @@
-# This is a shared configuration file that is intended to be stored in the project repo.
-# To override a variable locally:
-# - create .docksal/docksal-local.env file and local variable overrides there
-# - add .docksal/docksal-local.env to .gitignore
-
-# Use the default Docksal stack
+### Use the default Docksal stack
 DOCKSAL_STACK=default
 
-# Lock images versions for LAMP services
+### Lock images versions for LAMP services
 # This will prevent images from being updated when Docksal is updated
-#WEB_IMAGE='docksal/web:x.x-apache2.4'
-#DB_IMAGE='docksal/db:x.x-mysql-5.6'
-# Set for compatibility with contrib module that may not be ready for php 7.2
-CLI_IMAGE='docksal/cli:php7.1'
+# WEB_IMAGE='docksal/web:...'
+# DB_IMAGE='docksal/db:...'
+# CLI_IMAGE='docksal/cli:...'
+# See available versions at https://docs.docksal.io/stack/images-versions
 
-# Docksal configuration.
-#VIRTUAL_HOST=drupal7.docksal
-#DOCROOT=docroot
+### Docksal configuration.
+# VIRTUAL_HOST=drupal7.docksal
+# DOCROOT=docroot
 
 # MySQL settings.
 # MySQL will be exposed on a random port. Use "fin ps" to check the port.
 # To have a static MySQL port assigned, copy the line below into the .docksal/docksal-local.env file
 # and replace the host port "0" with a unique host port number (e.g. MYSQL_PORT_MAPPING='33061:3306')
-MYSQL_PORT_MAPPING='0:3306'
+# MYSQL_PORT_MAPPING='33061:3306'
 
 # Enable/disable xdebug
 # To override locally, copy the two lines below into .docksal/docksal-local.env and adjust as necessary
-XDEBUG_ENABLED=0
```

Comment:
Same here, wording above needs to be updated.
Maybe, just say " To enable xdebug locally, run `fin config set --env=local XDEBUG_ENABLED=1` then `fin project start`. Users would still need to know that `docksal-local.env` has to be gitignored (covered in the very first original paragraph).